### PR TITLE
Use Lift plugin in `aws-*-sqs-worker` examples

### PR DESCRIPTION
**Pull request by pgrzesik**
Use Lift plugin to simplify `aws-node-sqs-worker` and `aws-python-sqs-worker` examples

### CODE REVIEW

**Comment by Pigius** [comment]
Nice job @pgrzesik !

**Comment by pgrzesik** [comment]
Thanks @Pigius :bow:

File: aws-python-sqs-worker/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ serverless deploy
 
 After running deploy, you should see output similar to:
 
-```bash
+``` bash
```
Comment (medikoo) [inline aws-python-sqs-worker/README.md:41]: I think this change is not intended

File: aws-node-sqs-worker/README.md [hunk 2]
```diff
@@ -12,13 +12,11 @@ authorAvatar: 'https://avatars1.githubusercontent.com/u/13742415?s=200&v=4'
 
 # Serverless Framework Node SQS Producer-Consumer on AWS
 
-This template demonstrates how to develop and deploy a simple SQS-based producer-consumer service running on AWS Lambda using the traditional Serverless Framework. It allows to accept messages, for which computation might be time or resource intensive, and offload their processing to an asynchronous background process for a faster and more resilient system.
+This template demonstrates how to develop and deploy a simple SQS-based producer-consumer service running on AWS Lambda using the traditional Serverless Framework and Serverless Lift plugin. It allows to accept messages, for which computation might be time or resource intensive, and offload their processing to an asynchronous background process for a faster and more resilient system.
```
Comment (medikoo) [inline aws-node-sqs-worker/README.md:15]: It might be nice to link to Lift plugin

**Review by medikoo — COMMENTED** [review]
@pgrzesik looks great! I have just two comments

File: aws-python-sqs-worker/README.md (hunk 1 above)
Comment (pgrzesik) [reply aws-python-sqs-worker/README.md]: yes, good catch :+1:

File: aws-node-sqs-worker/README.md (hunk 2 above)
Comment (pgrzesik) [reply aws-node-sqs-worker/README.md]: Good suggestion - I've linked to `queue` docs but linking to Lift plugin at the beginning is a great call - changed

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**